docs/windows.md

```diff
@@ -426,6 +368,130 @@ antrea-agent-windows-6hvkw                             1/1     Running     0
+work with Antrea. You should skip this step on the kubernetes cluster starting
+from 1.26 and enable Antrea proxy `proxyAll` mode for Windows agent.
+
+Add Windows-compatible versions of kube-proxy by applying `kube-proxy.yaml`.
+Download `kube-proxy.yaml` from the Kubernetes official repository and set
+the kube-proxy version.
+
+An example command that downloads and applies `kube-proxy.yml` is like below:
+
+```bash
+curl -L "https://github.com/kubernetes-sigs/sig-windows-tools/releases/download/v0.1.5/kube-proxy.yml" | sed 's/VERSION-nanoserver/v1.20.0/g' > kube-proxy.yml
+```
+
+Replace the content of `run-script.ps1` in the `kube-proxy-windows` ConfigMap
+with following:
+
+```yaml
+apiVersion: v1
+data:
+  run-script.ps1: |-
+    $ErrorActionPreference = "Stop";
+    mkdir -force /host/var/lib/kube-proxy/var/run/secrets/kubernetes.io/serviceaccount
+    mkdir -force /host/k/kube-proxy
+
+    cp -force /k/kube-proxy/* /host/k/kube-proxy
+    cp -force /var/lib/kube-proxy/* /host/var/lib/kube-proxy
+    cp -force /var/run/secrets/kubernetes.io/serviceaccount/* /host/var/lib/kube-proxy/var/run/secrets/kubernetes.io/serviceaccount
+
+    wins cli process run --path /k/kube-proxy/kube-proxy.exe --args "--v=3 --config=/var/lib/kube-proxy/config.conf --proxy-mode=userspace --hostname-override=$env:NODE_NAME"
+
+kind: ConfigMap
+metadata:
+  labels:
+    app: kube-proxy
+  name: kube-proxy-windows
+  namespace: kube-system
+```
+
+For containerd runtime, replace the content of `run-script.ps1` with following:
+
+```yaml
+apiVersion: v1
+data:
+  run-script.ps1: |-
+    $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
+    $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/') 
+    New-Item -Path "c:/var/lib" -Name "kube-proxy" -ItemType "directory" -Force
+    ((Get-Content -path $mountPath/var/lib/kube-proxy/kubeconfig.conf -Raw) -replace '/var',"$($mountPath)/var") | Set-Content -Path /var/lib/kube-proxy/kubeconfig.conf
+    ((Get-Content -path /var/lib/kube-proxy/kubeconfig.conf -Raw) -replace '\/',"/") | Set-Content -Path /var/lib/kube-proxy/kubeconfig.conf
+    sed -i 's/mode: iptables/mode: \"\"/g' $mountPath/var/lib/kube-proxy/config.conf
+    & "$mountPath/k/kube-proxy/kube-proxy.exe" --config=$mountPath/var/lib/kube-proxy/config.conf --v=10 --proxy-mode=userspace --hostname-override=$env:NODE_NAME
+kind: ConfigMap
+metadata:
+  labels:
+    app: kube-proxy
+  name: kube-proxy-windows
+  namespace: kube-system
+```
+
+Set the `hostNetwork` option to `true` in spec of kube-proxy-windows daemonset.
+
+```yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    k8s-app: kube-proxy
+  name: kube-proxy-windows
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      k8s-app: kube-proxy-windows
+  template:
+    metadata:
+      labels:
+        k8s-app: kube-proxy-windows
+    spec:
+      hostNetwork: true
+```
+
+For containerd runtime, Set `hostNetwork` to `true` and add options in spec and command.
```

Comment:
"For containerd runtime, also add the following to the kube-proxy-windows DaemonSet spec"?

Reply:
Updated.